app/src/main/java/com/yacgroup/yacguide/TourbookActivity.kt

```diff
@@ -309,17 +340,14 @@ class TourbookActivity : BaseNavigationActivity() {
         startActivityForResult(intent, IntentConstants.REQUEST_SAVE_TOURBOOK)
     }
 
-    private fun _initYears(resetCurrentYear: Boolean = true) {
+    private fun _initYears(): Int {
```

Comment:
I think the method name no longer match the implemented function.

Reply:
I believe, it's the other way around :) That it didn't match before. However, I'll try to find a better one.